webservices/legal_docs/index_management.py

```diff
@@ -418,6 +420,8 @@
     "analysis": {"analyzer": {"default": {"type": "english"}}}
 }
 
+BACKUP_REPOSITORY_NAME = "legal_s3_repository"
```

Comment:
should we name the backup repository different that the original? something like this `legal_s3_repository_bkup`. so we can distinguish between those two repos.

Reply:
@pkfec Thanks for taking a look! I figured we'd keep only one backup repository - were you thinking it would be better to have two?

Reply:
@lbeaufort Not necessarily two back ups. I think one is good enough. I was thinking that giving a different name to the backup repository will make it easy to distinguish the repos,  that we have taken a back up.

Reply:
"most_recent_snapshot_name = get_most_recent_snapshot()
    if not snapshot_name:
        snapshot_name = most_recent_snapshot_name"
could be refactored to something like this for performance consideration:
snapshot_name = snapshot_name or get_most_recent_snapshot()

Reply:
Thanks @qqss88! Since I need it later for the error message, I changed it to :
```
    most_recent_snapshot_name = get_most_recent_snapshot()
    snapshot_name = snapshot_name or most_recent_snapshot_name
...
        logger.info(
            "You may want to try the most recent snapshot: {0}".format(
                most_recent_snapshot_name
            )
        )
```

Reply:
@pkfec I think we only take backups of the `docs` index in this repo, so we should be able to keep all the backups together in the existing repository.

Reply:
@lbeaufort I see your point. I am OK with your changes. You did an amazing work!